tests/src/main/scala/sttp/tapir/tests/package.scala

```diff
@@ -309,6 +315,50 @@ package object tests {
       .out(stringBody)
       .name("Query with default")
 
+  object MultipleMediaTypes {
+    implicit val schemaForPerson: Schema[Person] = Schema[Person](
+      SchemaType.SProduct(
+        SObjectInfo("sttp.tapir.tests.Person"),
+        List(FieldName("name") -> Schema(SString), FieldName("age") -> Schema(SInteger))
+      )
+    )
+
+    implicit val schemaForOrganization: Schema[Organization] = Schema[Organization](
+      SchemaType.SProduct(
+        SObjectInfo("sttp.tapir.tests.Organization"),
+        List(FieldName("name") -> Schema(SString))
+      )
+    )
+
+    implicit val xmlCodecForOrganization: XmlCodec[Organization] =
+      Codec.xml(_rawDecode = xml => DecodeResult.Value(
+        // <name>xxx</name>
+        Organization(xml.split(">")(1).split("<").head)
```

Comment:
nice parser :D

Reply:
does its job :D used scala xml module at first but it's not convertible to ScalaJS